templates/react/package.json

```diff
@@ -1,11 +1,17 @@
 {
-  "name": "{{ .Name }}",
+  "name": "commit0",
```

Comment:
this file is only used for local dev, see corresponding template file that is used in code generation